uw-frame-components/portal/main.js

```diff
@@ -152,6 +153,10 @@ define([
                 defaultThemeGo();
               }
               loadingCompleteSequence();
+            }, function(reason){
+              console.error("We got a error back from groupURL, setting theme to default");
```

Comment:
Gotcha. It would be nice if this were standard, this is a rare case that you would actually want an error in the log somewhere, ya know?